src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqExpectationsRepository.kt

```diff
@@ -104,17 +105,24 @@ class JooqExpectationsRepository(dsl: DSLContext)
 
         val outcomes = records.groupBy{ it[BURDEN_ESTIMATE_EXPECTATION.ID] }
             .mapValues{
-                it.value.mapNotNull{ row -> row[Tables.outcomes.OUTCOME] }
+                it.value.mapNotNull{ row -> row[Tables.outcomes.OUTCOME] }.distinct()
```

Comment:
Added deduping for both outcomes and scenarios and we have rows for each combination of outcome and scenario..